code/datums/armor.dm

```diff
@@ -1,97 +1,53 @@
-#define ARMORID "armor-[tier]-[linemelee]-[linebullet]-[linelaser]-[melee]-[bullet]-[laser]-[energy]-[bomb]-[bio]-[rad]-[fire]-[acid]-[magic]-[wound]"
```

Comment:
Removal of energy, fire, magic, and ***wound*** damage? The combination of energy and fire into laser is fine, although I'd suggest keeping the common name as 'fire' or 'burn', but wound armor is important.